src/qforte/qkd/nt_srqk.py

```diff
@@ -60,6 +61,8 @@ def run(self,
         self._n_cnot = 0
         self._n_pauli_trm_measures = 0
 
+        self._save_qk_matrices = save_qk_matrices
```

Comment:
My understanding is that `nt_srqk` is called before the MRSQK. Is there a risk of MRSQK overwriting the saved QK files from here?